packages/boilerplate/theme/nuxt.config.js

```diff
@@ -61,6 +61,7 @@ export default {
     // @core-development-only-start
     ['@vue-storefront/nuxt-theme', {
       generate: {
+        ignore: ['./theme/components/FilterSidebar.vue'],
```

Comment:
is this applied in `theme-utilities` ?

Reply:
No, should I add the `theme-utils.config.js` file to the `theme-utilities` package, with this code: 
```js
module.exports = {
  copy: {
    to: '',
    from: [
      {
        path: '',
        ignore: ['./theme/components/FilterSidebar.vue'],
        variables: {},
        watch: true
      }
    ]
  }
};
```